frb_example/pure_dart/rust/src/api.rs

```diff
@@ -958,36 +962,56 @@ pub fn nested_id(id: [TestId; 4]) -> [TestId; 2] {
     }
 }
 
-/// Opaque types
+pub fn sync_dart_opaque(not_temp: DartOpaque) -> SyncReturn<String> {
+    SyncReturn("test".to_owned())
+}
+
+pub fn async_dart_opaque(not_temp: DartOpaque) -> String {
+    "async test".to_owned()
+}
+
+pub fn loop_back(not_temp: DartOpaque) -> DartOpaque {
+    not_temp
+}
+
+pub fn exotic_drop(not_temp: DartOpaque) -> SyncReturn<i32> {
+    std::thread::spawn(move || {
+        drop(not_temp);
```

Comment:
What about adding a test that `sleep` for a second before dropping, so we double test the timing as well

Reply:
here we have to use `#[cfg target]` because `sleep` and `thread::spawn` are not web.

Reply:
Then what about a dead loop + check time until it passes one second, i.e. "busy wait".

Btw you are already using spawn, so I guess adding sleep is not a problem, if *both* are not web?